modules/person.rb

```diff
@@ -0,0 +1,45 @@
+# top level comment for linter error
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def to_get_id
+    id
+  end
+
+  def to_get_name
+    @name
+  end
+
+  def to_get_age
+    @age
+  end
+
+  def to_set_name=(name)
+    @name = name
+  end
+
+  def to_set_age=(age)
+    @age = age
+  end
+
+  def of_age
+    case age
+    when age >= 18
+      true
+    else
+      false
+    end
+  end
+
+  def can_use_services
+    return true if of_age? || parent_permission
+  end
```

Comment:
- [ ] It is often the convention in Ruby to add "?" to method names if they return boolean or truth values. Kindly add it to this method name: `can_use_services?`.

- [ ] We can also save some code here and we should use `@parent_permission`, not `parent_permission`

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```